application-tour-ui/src/main/resources/TourCode/TourJson.xml

```diff
@@ -132,6 +147,5 @@
   #set ($discard = $response.setContentType('application/json'))
   $jsontool.serialize({'tours': $tours})
 #end
-{{/velocity}}
-</content>
+{{/velocity}}</content>
```

Comment:
Yes, `git add -i` seems to do what I want. I wasn't aware of it, but I will definitely do that from now on. Sorry if I caused any trouble, I'm still new to XWiki application development :)